app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +349,45 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  parseSpeciesMetricDecimalPlaces = species => {
+    let parsedSpeciesMetrics = Object.assign({}, species);
+    let metricDecimalPlaces = {
+      agg_score: 0,
+      z_score: 1,
+      rpm: 1,
+      count: 0,
+      contigs: 0,
+      contig_r: 0,
+      percent_identity: 1,
+      alignment_length: 1,
+      e_value: 1,
+    };
+
+    Object.keys(species).forEach(outerMetric => {
+      if (outerMetric in metricDecimalPlaces) {
```

Comment:
When is this condition true? We should always have either nt or nr...

Reply:
This is true if the metric is not in nr or nt and is in METRIC_DECIMAL_PLACES (which is a metric that we want to adjust the decimal places)

Reply:
Here is the structure of a species:
`agg_score: "19195"
category: "bacteria"
common_name: ""
genus_tax_id: 1827
max_z_score: 6.475747921182336
name: "Rhodococcus erythropolis"
nr:
alignment_length: "49.0"
bg_mean: "40"
bg_stdev: "131"
count: "1"
e_value: "21.0"
percent_identity: "100.0"
rpm: "891.3"
z_score: "6.5"
__proto__: Object
nt:
alignment_length: "0.0"
bg_mean: "10"
bg_stdev: "31"
count: "0"
e_value: "0.0"
percent_identity: "0.0"
rpm: "0.0"
z_score: "-100.0"`

Reply:
I see, forgot about the aggregate score.